src/ProductConstructionService/ProductConstructionService.DependencyFlow/WorkItems/ActorWorkItem.cs

```diff
@@ -11,5 +11,5 @@ namespace ProductConstructionService.DependencyFlow.WorkItems;
 public abstract class ActorWorkItem : WorkItem
 {
     [DataMember]
-    public string ActorId { get; set; }
+    public required string ActorId { get; set; }
```

Comment:
```suggestion
    public required string ActorId { get; init; }
```
we don't want these to be mutable I think

Reply:
We don't but I need to try if the serialization to Redis works well